Tasks/PackerBuildV0/src/constants.ts

```diff
@@ -40,19 +47,30 @@ export var TemplateVariableClientSecretName = "client_secret";
 export var TemplateVariableTenantIdName = "tenant_id";
 export var TemplateVariableObjectIdName = "object_id";
 export var TemplateVariableSkipCleanName = "skip_clean";
+export var TemplateVariableManagedImageName = "managed_image_name"
 
 export var PackerLogTokenImageUri = "OSDiskUri";
 export var PackerLogTokenStorageLocation = "StorageAccountLocation";
+export var PackerLogTokenManagedResourceGroupName = "ManagedImageResourceGroupName";
+export var PackerLogTokenManagedImageName = "ManagedImageName";
+export var PackerLogTokenManagedImageLocation = "ManagedImageLocation";
 
 export var OutputVariableImageUri = "imageUri";
 
-export var CurrentSupportedPackerVersionString = "0.12.3";
-export var PackerDownloadUrlFormat =  "https://releases.hashicorp.com/packer/%s/packer_%s_%s.zip"
+export var CurrentSupportedPackerVersionString = "1.2.4";
```

Comment:
You are updating packer version here - is backward compat taken care of? means will an existing template which works with 0.12.3 packer will still work with 1.2.4 packer?

Reply:
Asking as this is a major version update for packer - they might have breaking change. Is it better to support both versions in task so that our existing customers do not break?

Reply:
Yes, or may be a packer tool installer which will install packer in a separate task. Let me discuss this with @azooinmyluggage

Reply:
Discussed with @azooinmyluggage and we will go ahead with with updating packer version to 1.2.4 in the task.
But you should bump up the major version for this task -  this will ensure that customer using old version will not be broken.

Reply:
In case you are not aware, for a new major version, you will need to create a new folder called PackerBuildV1 under Tasks folder and copy existing folder to there. Keep PackerBuildV0 to the same state it was earlier.
And add a new entry in make-options.json file

Reply:
Ok, will first resolve the merge conflicts and then make the other changes including creating a new major version.